config/locales/en/en.yml

```diff
@@ -117,6 +117,8 @@ en:
         actual_end_date: "Completed Date"
         duration: "Scheduled Duration"
         is_finalized: "Finalized"
+        old_duration_days: "Old Duration"
+        old_start_date: "Old Start Date"
```

Comment:
I think for display purposes, `Original` is better that `Old` unless you've been told otherwise by Roxana/client :)